src/scss/includes/direction.scss

```diff
@@ -76,11 +78,12 @@
 }
 
 .itinerary_leg_summary {
-  padding: $spacing-l;
+  padding: 0 $spacing-s;
```

Comment:
Why this change? As a result, the button is too close to the bottom of the screen and two consecutive summaries are too close when the panel is deployed.

![Capture d’écran de 2021-01-15 10-13-10](https://user-images.githubusercontent.com/243653/104705762-9bd31400-571a-11eb-8fd9-6b93bee9ee0e.png)

![Capture d’écran de 2021-01-15 10-13-18](https://user-images.githubusercontent.com/243653/104705787-a097c800-571a-11eb-8af5-25dce17547e8.png)